integration-tests/integration/prisonStatistics.spec.js

```diff
@@ -1,6 +1,11 @@
 import moment from 'moment'
 import { switchToIsoDateFormat } from '../../src/stringUtils'
 
+const PrisonStatsPage = require('../pages/prisonStatsPage')
+const KeyworkerPrisonStatsResponse = require('../responses/keyworkerPrisonStatsResponse').keyworkerPrisonStatsResponse
```

Comment:
Maybe you can use the `{KeyworkerPrisonStatsResponse, KeyworkerPrisonStatsNoCurrentDataResponse} = require('../responses/keyworkerPrisonStatsResponse')` construct for these 2 lines?